### PR TITLE
Done Translating-reference-dom-elements.md

### PR DESCRIPTION
**Pull request by splimter**

### CODE REVIEW

File: content/docs/reference-dom-elements.md [hunk 1]
```diff
@@ -106,37 +106,35 @@ React will automatically append a "px" suffix to certain numeric inline style pr
...
 ```js
 <div tabIndex="-1" />      // Just like node.tabIndex DOM API
 <div className="Button" /> // Just like node.className DOM API
 <input readOnly={true} />  // Just like node.readOnly DOM API
 ```
 
-These props work similarly to the corresponding HTML attributes, with the exception of the special cases documented above.
-
-Some of the DOM attributes supported by React include:
+تعمل هذه الخاصيّات بشكل مماثل لخاصيّات HTML الموافقة لها باستثناء الحالات الخاصيّة التي وثقناها بالأعلى. تتضمّن بعض خاصيّات DOM المدعومة من React:
```
Comment (asantarissy) [inline content/docs/reference-dom-elements.md]: الحالات الخاصة

File: content/docs/reference-dom-elements.md [hunk 2]
```diff
@@ -14,27 +14,27 @@ redirect_from:
...
+استخدم الخاصيّة `className` لتحديد صنف CSS. ينطبق ذلك على جميع عناصر DOM و SVG الاعتيادية مثل ‎`<div>`‎ و `‎<a>‎`، وغيرها.
 
-If you use React with Web Components (which is uncommon), use the `class` attribute instead.
+إن كنت تستخدم React مع مكوّنات الويب (وهو أمرٌ غير شائع) فاستخدم الخاصيّة class بدلًا من ذلك.
 
 ### dangerouslySetInnerHTML {#dangerouslysetinnerhtml}
 
-`dangerouslySetInnerHTML` is React's replacement for using `innerHTML` in the browser DOM. In general, setting HTML from code is risky because it's easy to inadvertently expose your users to a [cross-site scripting (XSS)](https://en.wikipedia.org/wiki/Cross-site_scripting) attack. So, you can set HTML directly from React, but you have to type out `dangerouslySetInnerHTML` and pass an object with a `__html` key, to remind yourself that it's dangerous. For example:
+الخاصيّة `dangerouslySetInnerHTML` هي بديل React لاستخدام `innerHTML` في DOM المتصفح. يكون تعيين HTML من الشيفرة أمرًا خطيرًا بشكلٍ عام لأنّه من السهل تعريق مستخدميك إلى [هجمات (XSS)](https://en.wikipedia.org/wiki/Cross-site_scripting) attack `dangerouslySetInnerHTML`وتمرير كائن مع المفتاح `__html` key, لتذكير نفسك بخطر فعل ذلك. على سبيل المثال:
```
[the remark this answers is not in the document]
Comment (splimter) [reply content/docs/reference-dom-elements.md]: i did fellow the hassoub ref and i did notice that he did mm delete some parts

[the remark this answers is not in the document]
Comment (asantarissy) [reply content/docs/reference-dom-elements.md]: Thanks @splimter, maybe they missed it, please review your translations as Hsoub might have some typos in their website.

File: content/docs/reference-dom-elements.md [hunk 3]
```diff
@@ -48,23 +48,23 @@ function MyComponent() {
...
+الخاصيّة `selected` مدعومة من قبل المكوّن ‎`<option>‎`. تستطيع استخدامها لتحديد ما إذا كان المكوّن مُحدَّدًا. يُفيد هذا لبناء مكوّنات مضبوطة.
 
 ### style {#style}
 
->Note
+>ملاحظة
 >
->Some examples in the documentation use `style` for convenience, but **using the `style` attribute as the primary means of styling elements is generally not recommended.** In most cases, [`className`](#classname) should be used to reference classes defined in an external CSS stylesheet. `style` is most often used in React applications to add dynamically-computed styles at render time. See also [FAQ: Styling and CSS](/docs/faq-styling.html).
+> تستخدم بعض الأمثلة في التوثيق الخاصيّة `style` للسهولة، ولكنّ استخدام الخاصيّة `‎<option>‎` كوسيلة أساسية لتنسيق العناصر هو أمر غير مفضّل بشكلٍ عام. يجب في معظم الحالات استخدام الخاصيّة `className` للإشارة إلى الأصناف المعرّفة في ملف تنسيق `CSS` خارجي. تُستخدَم الخاصيّة `style` غالبًا في تطبيقات React لإضافة تنسيقات محسوبة بشكل ديناميكي في الزمن الحقيقي. انظر أيضًا: [الأسئلة الأكثر شيوعًا: التنسيق و CSS](/docs/faq-styling.html).
```
[the remark this answers is not in the document]
Comment (splimter) [reply content/docs/reference-dom-elements.md]: Please Check this am not sure if i did fix it well

**Comment by splimter** [comment]
Check the new fixes <3

**Pull request merged**